### ISSUE TITLE
Broken repo build

### ISSUE THREAD

**Issue by moltco** [microsoft/WinUI-Gallery#1659]
### Which version of the app?

WinUI 3 Gallery

### Description

I cloned this repo using VS 2022 17.12 Community, opened DesktopWrapper.sln and tried to compile/run the 'Packaged' target as per the readme.md instructions. 

The build didn't work. It is broken. It feels dependencies and .csproj files in the solution are at fault. 

I get the following errors when trying to build the 'packaged' target:
```
The feed 'MUX-Dependencies [https://pkgs.dev.azure.com/shine-oss/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json]' lists package 'Microsoft.WindowsAppSDK.1.3.230331000' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again.
```
```
The feed 'MUX-Dependencies [https://pkgs.dev.azure.com/shine-oss/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json]' lists package 'Microsoft.WindowsAppSDK.1.3.230331000' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again.
```
```
The runtime pack for Microsoft.Windows.SDK.NET.Ref was not downloaded. Try running a NuGet restore with the RuntimeIdentifier 'any'.
```

I tried restoring nuget with runtime identifier any and trying to update nuget packages - it all failed, I guess the .csproj config is preventing nuget from working properly. 

Ultimately I am trying to test  MediaPlayerElement examples but I can't see a way to test/compile just those examples without the entire repo.

Could anyone help with resolving the build issues until there is a proper fix?

### Windows version

Windows 11 24H2 (26052)

**Comment by czy-29** [comment]
I have encountered the same issue here!

**Comment by AndrewKeepCoding** [comment]
Have you both tried opening *WinUIGallery.sln*?

**Comment by czy-29** [comment]
> Have you both tried opening _WinUIGallery.sln_?

Yes, but later I deleted the entire repository and cloned it again, and then opened the correct sln file, but the problem still exists.

**Comment by moltco** [comment]
@AndrewKeepCoding can you clone from scratch and compile with vs 17.12 without issues?

**Comment by AndrewKeepCoding** [comment]
@moltco I just did it to reconfirm it and works (compiles and runs) both *WinUIGallery.sln* and *WinUIGallery.DesktopWap.sln*. 🤔

**Comment by AndrewKeepCoding** [comment]
For some reason you might have issues downloading packages from:
https://[pkgs.dev.azure.com/shine-oss/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json](https://pkgs.dev.azure.com/shine-oss/microsoft-ui-xaml/_packaging/MUX-Dependencies/nuget/v3/index.json)

**Comment by moltco** [comment]
That's what I thought too @AndrewKeepCoding  but I can open the link in browser and have no custom nuget or VS configuration so I thought maybe that's a link to a private Microsoft nugetbor somesuch. 

Will try installing manually with nuget to see where it gets me.

**Comment by Dealman** [comment]
Same issue here using Visual Studio 17.12.1. Can't even download it manually using nuget, getting this error;
`NotFound https://pkgs.dev.azure.com/shine-oss/4ac19aec-8963-4452-80e5-70636e9ee157/_packaging/af27d749-fc85-4439-9fe3-94ec321597a3/nuget/v3/flat2/microsoft.windowsappsdk/1.3.230331000/microsoft.windowsappsdk.1.3.230331000.nupkg 861ms`

I noticed if I try and go to that URL manually via my browser, it opens a log-in prompt. Whereas the other `nupkg` URLs do not. Possibly a server-sided misconfiguration?

**Comment by czy-29** [comment]
> Same issue here using Visual Studio 17.12.1. Can't even download it manually using nuget, getting this error; `NotFound https://pkgs.dev.azure.com/shine-oss/4ac19aec-8963-4452-80e5-70636e9ee157/_packaging/af27d749-fc85-4439-9fe3-94ec321597a3/nuget/v3/flat2/microsoft.windowsappsdk/1.3.230331000/microsoft.windowsappsdk.1.3.230331000.nupkg 861ms`
> 
> I noticed if I try and go to that URL manually via my browser, it opens a log-in prompt. Whereas the other `nupkg` URLs do not. Possibly a server-sided misconfiguration?

Facing the same thing!

**Comment by marcelwgn** [comment]
I am able to reproduce this, I needed to delete my locally NuGet cache for this error to occur though (located under `C:\Users\[user]\.nuget\packages`.

As a workaround for now, you can delete the `nuget.config` and you should be able to build the project correctly.

**Comment by czy-29** [comment]
> I am able to reproduce this, I needed to delete my locally NuGet cache for this error to occur though (located under `C:\Users\[user]\.nuget\packages`.
> 
> As a workaround for now, you can delete the `nuget.config` and you should be able to build the project correctly.

Okay, after deleting `nuget.config`, I can successfully build it. Thank you!

**Comment by marcelwgn** [comment]
> > Okay, after deleting `nuget.config`, I can successfully build it. Thank you!
> 
> Simple solution. I do expect an "official" solution, because I just had the same issue.

This issue is definitely on our radar :)

**Comment by Zakariathr22** [comment]
I think this is related to the changes made to [nuget.config](https://github.com/microsoft/WinUI-Gallery/blob/main/nuget.config) in #1636
![Image](https://github.com/user-attachments/assets/91e605c7-7281-4713-a086-affc2900a3e0)

**Comment by manabu-nakamura** [comment]
> I still need to delete it 🤔

Thank you for letting me know. It seemed that the latest WinUI 3 Gallery can be built by Visual Studio 2022 17.14.2 after deleting `C:\Users\...\.nuget` and cloning it (without deleting `nuget.config`).

**Comment by Jay-o-Way** [comment]
Ping
